### ISSUE TITLE
GASP table adjustment

### ISSUE THREAD

**Issue by aaronbell** [source-foundry/dehinter#38]
At current, the GASP table modification appears to change the rangeGaspBehavior to 0x000f, which turns on:
-gridfit
-grayscale
-symmetric gridfit
-symmetric smoothing

My understanding is that this turns on hinting at the full ppem range (with smoothing, and all that). 

For a script that is meant to remove all hinting, shouldn't the rangeGaspBehavior be set to 0x000a, which is:
-grayscale
-symmetric smoothing

This way you have full smoothing and the rasterizer doesn't try to apply gridfit when no hinting is present to instruct it how to implement gridfit.

**Comment by chrissimpkins** [comment]
Thanks for the report! 

I don't recall the approach here off the top of my head but I attempted to RE `ttfautohint -d` and may well have missed some parts of the dehinting approach.  Do you know how ttfa -d handles it?

**Comment by aaronbell** [comment]
It appears that `ttfautohint -d` modifies the GASP table to match what your code does. However, I suspect that this not because it is making an intelligent decision about what is best for an unhinted font, and instead because it always sets the GASP table when tthautohint runs.

In the `ttfautohint` code, I found:
```
The font's `gasp`
 *     table is set up to always use grayscale rendering with grid-fitting
 *     for standard hinting, and symmetric grid-fitting and symmetric
 *     smoothing for horizontal subpixel hinting (ClearType).
```

I could not find any references anywhere to GASP table modifications in relation to dehinting. 

In a discussion that I had with Mike Duggan a year or so ago, "when not hinting, we want the GASP set to use full ysmoothing and no hinting." Given that, I think that you probably only want bit 2 and 4, thus disabling any gridfit.

**Comment by chrissimpkins** [comment]
Agree! Let's do the right thing.  Interested in submitting a PR?

**Comment by aaronbell** [comment]
Done!

**Comment by chrissimpkins** [comment]
Ty!  I'll update the CI tests and get this merged.  Much appreciated!!

**Comment by chrissimpkins** [comment]
changes merged to the master branch and will be released in dehinter v2.0.0
